keyboards/sneakbox/lilbae/keyboard.json

```
@@ -0,0 +1,66 @@
+{
+    "manufacturer": "Sneakbox",
+    "maintainer": "mujimanic",
+    "bootloader": "atmel-dfu",
+    "build": {
+        "lto": true
+    },
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "D2", "pin_b": "D3"}
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "encoder": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["B7", "D6", "D5", "D4"],
+        "rows": ["B4"]
+    },
+    "processor": "atmega32u2",
+    "url": "https://sneakbox.com",
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x0012",
+        "vid": "0x5342"
+    },
+    "layouts": {
+        "LAYOUT_all": {
+            "layout": [
+                {"matrix": [0, 0], "x": 1.75, "y": 0, "w": 1.5},
+                {"matrix": [0, 1], "x": 2, "y": 1, "w": 1.25},
+                {"matrix": [0, 2], "x": 1, "y": 1},
+                {"matrix": [0, 3], "x": 0, "y": 1}
+            ]
+        },
+        "LAYOUT_ansi_enter": {
+            "layout": [
+                {"matrix": [0, 0], "x": 0.75, "y": 0, "w": 1.5},
+                {"matrix": [0, 2], "x": 0, "y": 1, "w": 2.25}
+            ]
+        },
+        "LAYOUT_bae": {
+            "layout": [
+                {"matrix": [0, 2], "x": 0, "y": 0, "w": 2.25, "h": 2}
```

```suggestion
                {"matrix": [0, 2], "x": 0.75, "y": 0, "w": 1.5, "h": 2}
```
render as bae in configurator